Include/internal/pycore_ceval_state.h

```diff
@@ -13,6 +13,8 @@ extern "C" {
 #include "pycore_gil.h"             // struct _gil_runtime_state
 
 
+typedef int (*_Py_pending_call_func)(void *);
```

Comment:
I don't think this refactoring should be done in 3.12.1. Is there a reason not to leave it just 'int (*func)(void *)' in a few places?

Reply:
Yeah, `_Py_pending_call_func()` is not necessary.  I'm fine with dropping it.